With version  2.0.2 getting react-native-quick-base64:generateJsonModelDebug error

Hi,

with the update 2.0.2 android build is failing with the following error. Can you please check

Execution failed for task ':react-native-quick-base64:generateJsonModelDebug'.
> C:\Users\aj\work\node_modules\react-native-quick-base64\android\CMakeLists.txt : C/C++ debug|armeabi-v7a : CMake Error at C:\Users\aj\work\node_modules\react-native-quick-base64\android\CMakeLists.txt:14 (add_library):
    Syntax error in cmake code when parsing string

      C:\Users\aj\work\node_modules/react-native/ReactCommon/jsi/jsi/jsi.cpp

    Invalid character escape '\U'.

--- a/android/CMakeLists.txt
+++ b/android/CMakeLists.txt
@@ -4,6 +4,8 @@ set (PACKAGE_NAME "react-native-quick-base64")
 set (CMAKE_VERBOSE_MAKEFILE ON)
 set (CMAKE_CXX_STANDARD 11)
 
+file(TO_CMAKE_PATH "${NODE_MODULES_DIR}/react-native/ReactCommon/jsi/jsi/jsi.cpp" libPath)
+
 include_directories(
   ../cpp
   "${NODE_MODULES_DIR}/react-native/React"
@@ -13,7 +15,7 @@ include_directories(
 
 add_library(quickbase64 # Library name
   SHARED
-  "${NODE_MODULES_DIR}/react-native/ReactCommon/jsi/jsi/jsi.cpp"
+  ${libPath}
   ../cpp/base64.cpp
   ../cpp/base64.h
   ../cpp/react-native-quick-base64.cpp

